__tests__/components/AmountDisplay.spec.ts

```diff
@@ -33,14 +33,17 @@ describe('AmountDisplay', () => {
         amountDisplayTs = amountDisplay.vm as AmountDisplayTs;
     });
 
+    const localDecimalSeparator = Number(0.0).toFixed(1).toLocaleString().substring(1, 2);
+    const expectedFractionalPart = localDecimalSeparator + '234';
```

Comment:
can you hardcode expected string instead or is it variable?

Reply:
yes, it varies based on the user's locale. It wasn't too easy to mock the implementation of `Number.toLocaleString` in jest for a particular locale, so I left it dynamic.

But, after your comment, I found an alternative way and changed the implementation a little bit and passed the locale to the component as props for better testing.

Reply:
fixed